nifi-ngsi-bundle/nifi-ngsi-processors/pom.xml

```diff
@@ -87,7 +92,7 @@
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-processor-utils</artifactId>
-            <version>${nifi.version}</version>
+            <version>1.15.3</version>
```

Comment:
why did you remove `${nifi.version}` (which avoid repeating the NiFi version everywhere) and replace it with an old version??

Reply:
The nifi version `1.27.0 ` doesn't match for this dependency, the latest version available is `1.15.3`

Reply:
this is because it no longer exists after 1.15.3: https://issues.apache.org/jira/browse/NIFI-9610

so this dependency should be removed (and new ones added only if necessary).